src/utils/slack_notification.py

```diff
@@ -0,0 +1,57 @@
+import json
+import requests
+from config_key import SLACK_WEBHOOKS
```

Comment:
probably always import from `config` instead of `config_key` by importing settings from config_key in `config.py` (https://github.com/SmartAPI/smartAPI/blob/master/src/config.py#L5)